README.rst

```diff
@@ -15,13 +15,13 @@ Airtest
 Getting Started
 ---------------
 
-Airtest is a cross-platform automated testing framework focused mainly on games, but can also be used for native apps. Windows and Android are currently supported. iOS support will be coming in the near future.
+Airtest is a cross-platform automated testing framework focussing mainly on games, but can also be used for native apps. Windows and Android are currently supported; iOS support will be available in the near future.
```

Comment:
focusing?

Reply:
i think focusing is grammatically correct